app/views/games/show.html.erb

```diff
@@ -31,29 +31,21 @@
   };
 
   $(function() {
-    $('.chessboard').each(function() {
-      var $chessboard = $(this);
-      var piecesURL = $chessboard.data('pieces-url');
 
-      // populates boards with pieces
+    setupBoard();
+
+    function setupBoard() {
+      $('.piece').remove();
+      var piecesURL = $('.chessboard').data('pieces-url');
+
       $.get(piecesURL).success(function(pieces) {
         pieces.forEach(function(piece, i){
-          if (!piece.is_captured) {
+          if (piece.row > -1 && piece.col > -1) {
```

Comment:
That is very reasonable. However, @MarciaOrr wanted a front-end feature to display captured pieces to the side of the board. We would need to request all the pieces for that feature to work.